src/CTA.WebForms2Blazor/Constants.cs

```diff
@@ -66,6 +67,18 @@ internal class Constants
         public const string EventArgsParamTypeName = "EventArgs";
         public const string HttpContextParamTypeName = "HttpContext";
 
+        // View Text Templates
+        public const string MarkupCommentTemplate = "<!-- {0} -->";
+        public const string RazorServerSideCommentTemplate = "@* {0} *@";
+        public const string RazorCodeBlockEmbeddingTemplate = "@{{ {0} }}";
```

Comment:
Are the spaces here necessary? Like is it not supposed to match anything that is say "@{{{0}}}"?
Also might want to use regex whitespace character instead of just space

Reply:
Might've been a misunderstanding here, these templates are used for code insertion not code detection